src/tr2/game/objects/creatures/dragon.c

```diff
@@ -36,23 +36,23 @@
 
 typedef enum {
     // clang-format off
-    DRAGON_ANIM_EMPTY       = 0,
-    DRAGON_ANIM_WALK        = 1,
-    DRAGON_ANIM_LEFT        = 2,
-    DRAGON_ANIM_RIGHT       = 3,
-    DRAGON_ANIM_AIM         = 4,
-    DRAGON_ANIM_FIRE        = 5,
-    DRAGON_ANIM_STOP        = 6,
-    DRAGON_ANIM_TURN_LEFT   = 7,
-    DRAGON_ANIM_TURN_RIGHT  = 8,
-    DRAGON_ANIM_SWIPE_LEFT  = 9,
-    DRAGON_ANIM_SWIPE_RIGHT = 10,
-    DRAGON_ANIM_DEATH       = 11,
-    DRAGON_ANIM_DIE         = 21,
-    DRAGON_ANIM_DEAD        = 22,
-    DRAGON_ANIM_RESURRECT   = 23,
+    DRAGON_STATE_EMPTY       = 0,
+    DRAGON_STATE_WALK        = 1,
+    DRAGON_STATE_LEFT        = 2,
+    DRAGON_STATE_RIGHT       = 3,
+    DRAGON_STATE_AIM         = 4,
+    DRAGON_STATE_FIRE        = 5,
+    DRAGON_STATE_STOP        = 6,
+    DRAGON_STATE_TURN_LEFT   = 7,
+    DRAGON_STATE_TURN_RIGHT  = 8,
+    DRAGON_STATE_SWIPE_LEFT  = 9,
+    DRAGON_STATE_SWIPE_RIGHT = 10,
+    DRAGON_STATE_DEATH       = 11,
+    DRAGON_STATE_DIE         = 21,
```

Comment:
21-23 are anim indices (further below you can see they are compared with the current anim index rather than anim state. So maybe a separate anim enum, or fixed defines for these three would be an idea.

Reply:
@lahm86 addressed